src/scancode/cli.py

```diff
@@ -109,6 +113,13 @@
 
 notice = acknowledgment_text.strip().replace('  ', '')
 
+scan_proper_plugins = PluginManager('scan_proper')
+scan_output_plugins = PluginManager('scan_output')
+scan_proper_plugins.add_hookspecs(hookspec)
+scan_output_plugins.add_hookspecs(hookspec)
```

Comment:
ok, as commented above, there should be only one hook spec for a format plugin, not two. The spec defines what and where in the scan pipeline a plugin contributes. Not how this is done or separating adding options from adding actual feature.